ui/app/models/role-pki.js

```diff
@@ -55,6 +55,10 @@ export default DS.Model.extend({
     defaultValue: 'DigitalSignature,KeyAgreement,KeyEncipherment',
     editType: 'stringArray',
   }),
+  extKeyUsageOIDs: attr({
```

Comment:
ext_key_usage_oids will get "camelized" to `extKeyUsageOids` - so will need to change that here and below.